_targets.R

```diff
@@ -21,9 +27,16 @@ list(
   tar_target(oldest_active_sites, find_oldest_sites(states, parameter)),
 
   tar_map(
-    values = tibble(state_cd = states),
-    tar_target(nwis_inventory, get_state_inventory(oldest_active_sites, state_cd)),
-    tar_target(nwis_data, get_site_data(nwis_inventory, state_cd, parameter))
+    values = tibble(state_cd = states)%>%
+      mutate(state_plot_files = sprintf("3_visualize/out/timeseries_%s.png", state_cd)),
+    tar_target(nwis_inventory, filter(oldest_active_sites, state_cd == states)),
```

Comment:
I think you want to have your `filter()` statement use the static branching values in `state_cd` (one at a time), not the target `states` (all your states). So, you should change this to `filter(oldest_active_sites, state_cd == state_cd)`